style.css

```diff
@@ -121,3 +121,646 @@ a {
   color: #384bee;
   cursor: pointer;
 }
+
+#work {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  margin: 20px;
+  padding-top: 5rem;
+}
+
+#work .work-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1rem;
+  grid-gap: 10px;
+  width: 100%;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #def1e6;
+  border-radius: 1rem;
+}
+
+.work-card .img1 {
+  width: 100%;
+  height: 220px;
+  border-radius: 0.5rem;
+}
+
+.work-card h2 {
+  width: 100%;
+  height: 36px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 44px;
+  color: #172b4d;
+  margin-left: 0.2rem;
+}
+
+.work-card .align {
+  width: 100%;
+}
+
+.work-card .align ul {
+  color: #c1c7d0;
+  display: flex;
+  margin-top: 0.5rem;
+}
+
+.work-card .align span {
+  width: 55px;
+  height: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 1rem;
+  color: #344563;
+  margin-left: 0.4rem;
+}
+
+.work-card .align ul li {
+  width: 88px;
+  height: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 1rem;
+  color: #7a869a;
+  justify-content: space-between;
+  margin-left: 2rem;
+}
+
+.work-card p {
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-left: 0.5rem;
+}
+
+.tags {
+  width: 100%;
+}
+
+.tags .hash {
+  display: flex;
+  list-style: none;
+  padding-right: 4.5rem;
+  gap: 0.5rem;
+  margin-left: 0.2rem;
+}
+
+.tags .hash .HTML {
+  padding: 4px 12px;
+  width: 57px;
+  height: 1.5rem;
+  background: #ebebff;
+  border-radius: 0.5rem;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] - [ ] It is noticed that `font-family: 'Poppins', sans-serif;` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.primary-font` and assigning it wherever possible as a class name in your `html` markup code.
```
.primary-font {
  display: flex;
}
```
- And kindly attend to any repetitions in your code 🙂.
- Kindly note that DRY is a requirement, but since the other reviewers have been overlooking it, I've decided to mark it as optional. 😉
- You repeated this 28times 😄 
<img width="162" alt="image" src="https://user-images.githubusercontent.com/41163571/234948724-c92ed668-60a9-4ea1-b391-f41d75c5b593.png">